### PR TITLE
[chrome-api-jstocxxtest] Add GetStatusChange test

### PR DESCRIPTION
**Pull request by ViktoriiaKovalova**

### CODE REVIEW

File: smart_card_connector_app/src/testing-smart-card-simulation-constants.js [hunk 1]
```diff
@@ -41,4 +41,9 @@ GSC.TestingLibusbSmartCardSimulationConstants.COSMO_ID_70_ATR =
       0x3B, 0xDB, 0x96, 0x00, 0x80, 0xB1, 0xFE, 0x45, 0x1F, 0x83, 0x00,
       0x31, 0xC0, 0x64, 0xC7, 0xFC, 0x10, 0x00, 0x01, 0x90, 0x00, 0x74,
     ])).buffer;
+
+// The constant from the PC/SC-Lite API docs.
+/** @const */
+GSC.TestingLibusbSmartCardSimulationConstants.PNP_NOTIFICATION =
+    String.raw`\\?PnP?\Notification`;
```
Comment (emaxx-google) [inline smart_card_connector_app/src/testing-smart-card-simulation-constants.js:48]: This value is not really simulation-specific... Maybe put this into `third_party/pcsc-lite/naclport/js_client/src/api.js`?

Comment (emaxx-google) [reply smart_card_connector_app/src/testing-smart-card-simulation-constants.js:48]: Or perhaps `third_party/pcsc-lite/naclport/common/src/constants.js`.

Comment (ViktoriiaKovalova) [reply smart_card_connector_app/src/testing-smart-card-simulation-constants.js]: done

**Review by Fabian-Sommer — APPROVED** [review]
LGTM

**Pull request merged**